### PR TITLE
sql-plan-replayer: add sql-plan-replayer doc

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
sql-plan-replayer: add sql-plan-replayer doc

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/7380
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by rebelice — APPROVED** [review]

File: sql-plan-replayer.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+title: Use PLAN REPLAYER to Save and Restore the On-Site Information of a Cluster
+summary: Learn how to use PLAN REPLAYER to save and restore the on-site information of a cluster
```
Comment (en-jin19) [inline sql-plan-replayer.md:3]:
```suggestion
summary: Learn how to use PLAN REPLAYER to save and restore the on-site information of a cluster.
```

File: sql-plan-replayer.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+---
+
+# Use PLAN REPLAYER to Save and Restore the On-Site Information of a Cluster
+
+When you locate and troubleshoot the issues of a TiDB cluster, you often need to provide information on the system and the execution plan. To help you get the information and troubleshoot cluster issues in a more convenient and efficient way, the `PLAN REPLAY` command is introduced in TiDB v5.3.0. This command enables you to easily save and restore the on-site information of a cluster, improves the efficiency of troubleshooting, and helps you more easily archive the issue for management.
```
Comment (en-jin19) [inline sql-plan-replayer.md:8]:
```suggestion
When you locate and troubleshoot the issues of a TiDB cluster, you often need to provide information on the system and the execution plan. To help you get the information and troubleshoot cluster issues in a more convenient and efficient way, the `PLAN REPLAY` command is introduced in TiDB v5.3.0. This command enables you to easily save and restore the information about on-site issues, improves the efficiency of the execution plan that diagnoses issues, and helps you more easily archive and manage the issues.
```

File: sql-plan-replayer.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+The features of `PLAN REPLAYER` are as follows:
+
+- Exports the information of a TiDB cluster at an on-site troubleshooting to a ZIP-formatted file for storage.
+- Imports into a cluster the ZIP-formatted file exported from another TiDB cluster. This file contains the information of the latter TiDB cluster at an on-site troubleshooting.
```
Comment (en-jin19) [inline sql-plan-replayer.md:13]:
```suggestion
- Imports into a cluster a ZIP-formatted file exported from another TiDB cluster. This file contains the information of the latter TiDB cluster at an on-site troubleshooting.
```

File: sql-plan-replayer.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+curl http://127.0.0.1:10080/plan_replayer/dump/replayer_single_JOGvpu4t7dssySqJfTtS4A==_1635750890568691080.zip > plan_replayer.zip
+```
+
+## Use `PLAN REPLAYER` to import cluster information
+
+> **Warning:**
+>
+> When you import the on-site information of a TiDB cluster to another cluster, the TiDB session variables, SQL bindings, table schemas and statistics of the latter cluster are modified.
+
+With an existing `ZIP` file exported using `PLAN REPLAYER`, you can use the `PLAN REPLAYER` import interface to restore the on-site information of a cluster to any other TiDB cluster. The syntax is as follows:
```
Comment (en-jin19) [inline sql-plan-replayer.md:86]:
```suggestion
With an existing `ZIP` file exported using `PLAN REPLAYER`, you can use the `PLAN REPLAYER` import interface to restore the on-site information of a cluster to any other TiDB clusters. The syntax is as follows:
```

**Comment by en-jin19** [comment]
rest LGTM

**Review by en-jin19 — APPROVED** [review]
LGTM

**Review by qiancai — APPROVED** [review]

**Pull request merged**